website/docs/API.md

```diff
@@ -737,6 +782,9 @@ function configure(options: Partial<Config>)  {}
 
 Default timeout, in ms, for async helper functions (`waitFor`, `waitForElementToBeRemoved`) and `findBy*` queries. Defaults to 1000 ms.
 
+#### `defaultDebugOptions` option
+
+Default `debugOptions` to be used when calling `debug()`. These default options will be overridden by the ones you specify directly when calling `debug()`. Can be used namely to transform prop values for more readability or filter out props that have little value when debugging tests (e.g. svg's path prop).
```

Comment:
I would rather refer user to `debug` `options` docs for details rather than repeating the use cases.

```suggestion
Default `debugOptions` to be used when calling `debug()`. These default options will be overridden by the ones you specify directly when calling `debug()`. 
```

Reply:
agreed, I added a link to debug options doc